internal/diagnose/segment.go

```diff
@@ -0,0 +1,27 @@
+package diagnose
+
+import (
+	"github.com/newrelic/newrelic-cli/internal/config"
+	configAPI "github.com/newrelic/newrelic-cli/internal/config/api"
+	"github.com/newrelic/newrelic-cli/internal/install/recipes"
+	"github.com/newrelic/newrelic-cli/internal/segment"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/http/httpproxy"
+)
+
+func initSegment() *segment.Segment {
```

Comment:
It's nice that we now have a `segment` package, I wonder if we could consolidate the `initSegment` (and) functions inside `internal/diagnose/segment.go` and `internal/install/command.go` into one inside that new package to not duplicate that code...

A similar thought possibly for `IsProxyConfigured` too, which we have in `internal/install/proxy_configuration.go` and `internal/diagnose/segment.go`.

Reply:
Good call, moved these over to `segment`. This caused an import cycle at first, but I resolved that by extracting the write key read out of `install/recipes` and over to another embedded FS in `segment`